index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Parks</title>
+        <header>
+            <img src="https://i.pinimg.com/736x/e0/b3/56/e0b35696d26ad189cb3301364c1bada1.jpg" alt="Header Image">
+            <h1>Cider Falls</h1>
+        </header>
```

Comment:
Need to move header element down to first line inside body element i.e. header should start on line 14